docs/platform/linode-gpu/getting-started-with-gpu/index.md

```diff
@@ -113,26 +143,46 @@ Installation on CentOS and RHEL operating systems will fail following this as th
 +-----------------------------------------------------------------------------+
 {{< /output >}}
 
-    In the output above, we can see that our Driver is installed and functioning correctly, the version of CUDA attributed to it, and other useful statistics.
+    In the output above, we can see that our driver is installed and functioning correctly, the version of CUDA attributed to it, and other useful statistics.
 
 ## Manual Install
 
-This section will walk you through the process of downloading and installing the latest NVIDIA driver on Debian 9. This process can be completed similarly on a Distro of your choice if needed:
+This section will walk you through the process of downloading and installing the latest NVIDIA driver on Debian 9. This process can also be completed similarly on another distro of your choice if needed:
+
+1. Visit NVIDIA's [Driver Downloads Page](https://www.nvidia.com/Download/index.aspx?lang=en-us).
+
+1. Make sure that the options from the drop-down menus reflect the following values:
 
-1. Find the latest Driver from NVIDIA's [Driver Downloads Page](https://www.nvidia.com/Download/index.aspx?lang=en-us).
+    | Prompt | Selection |
+    |--------|-----------|
+    | Product Type | Quadro |
+    | Product Series | Quadro RTX Series |
+    | Product | Quadro RTX 8000 |
+    | Operating System | Linux 64-bit |
+    | Download Type | Optimal Driver for Enterprise (ODE) |
+    | Download Type | Linux Long Lived Driver |
+    | Language | English (US) |
 
-1. Make sure that the options from the drop-down menus reflect the following:
+    The form will look as follows when completed:
 
     ![Driver Download](driver.png "Driver Download")
 
-1. Select "Search", "Download", and download the Linux driver that appears on the following screen to your Linode.
+1. Click the **Search** button, and a page will appear that shows information about the driver. Click the green **Download** button on this page. The file will not download to your computer; instead, you will be taken to another download confirmation page.
```

Comment:
Revised the existing instructions and modeled them on the copy-link-and-wget instructions from the Installing with CUDA section

Reply:
Also, there was one extra download confirmation page on NVIDIA's site that the previous instructions didn't mention